### ISSUE TITLE
Virtual Network Editor not working

### ISSUE THREAD

**Issue by robertstrom** [bytium/vm-host-modules#2]
Hello,

I have applied the update for the VMware Workstation host modules and VMware Workstation itself appears to be working but the Virtual Network Editor is not.

Is this supposed to be functional?

Thanks,

Robert

**Comment by bytium** [comment]
> Hello,
> 
> I have applied the update for the VMware Workstation host modules and VMware Workstation itself appears to be working but the Virtual Network Editor is not.
> 
> Is this supposed to be functional?
> 
> Thanks,
> 
> Robert

It is working here in Debian testing fully updated.

Which distro and which version Vmware workstation are you using?

**Comment by bytium** [comment]
> Pop!_OS 22.04 (Ubuntu based)
> 
> I wonder if it has anything to do with the fact that I had applied the Workstation fix from here previously. This fix is known to have an issue with the Virtual Network editor - https://github.com/nan0desu/vmware-host-modules
> 
> ```
> └─(10:15:36)──> cat /etc/os-release                                                                               ──(Tue,Sep10)─┘
> NAME="Pop!_OS"
> VERSION="22.04 LTS"
> ID=pop
> ID_LIKE="ubuntu debian"
> PRETTY_NAME="Pop!_OS 22.04 LTS"
> VERSION_ID="22.04"
> HOME_URL="https://pop.system76.com"
> SUPPORT_URL="https://support.system76.com"
> BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
> PRIVACY_POLICY_URL="https://system76.com/privacy"
> VERSION_CODENAME=jammy
> UBUNTU_CODENAME=jammy
> LOGO=distributor-logo-pop-os
> ```

Let me test it on my VM. Will get back soon!

**Comment by bytium** [comment]
I tried in Pop OS. The virtual network editor GUI did not start. Authentication behaves weirdly. 

So, For confirmation, I tested this patch with Fedora, too, and it works flawlessly:
![image](https://github.com/user-attachments/assets/23c5f05d-753a-44b8-8d2a-a6e858332529)

Look, there are some permission issues when starting such a GUI.

**Comment by arnydo** [comment]
+1 for https://github.com/bytium/vm-host-modules/issues/2#issuecomment-2341564504

**Comment by robertstrom** [comment]
I have done some additional testing and did fresh installations of Pop!_OS 22.04, Fedora 40, Kali Linux and Elementary OS and installed VMware Workstation 17.6 on each of them (installing no other apps) and I was able to access the Virtual Network Editor in both of those other Linux Operating systems.

So, three other operating systems where I was able to launch the VMware Workstation Virtual Network Editor without issue but I am not able to launch it on Pop!_OS. All 4 operating systems were virgin installations, fully updated, then VMware Workstation installed and the kernel extension installed if necessary. No other applications were installed.
